templates/Alloy.Mvc/Docker/create-db.sh

```diff
@@ -1,12 +1,23 @@
 #!/bin/bash
 
+echo "Creating database..."
+
+let result=1
+
 for i in {1..100}; do
-    /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P "$SA_PASSWORD" -Q "IF NOT EXISTS (SELECT * FROM sys.databases WHERE name = '${DB_NAME}') CREATE DATABASE ${DB_NAME} ON (NAME=${DB_NAME}_data, FILENAME='/var/opt/mssql/host_data/${DB_NAME}.mdf') LOG ON (NAME=${DB_NAME}_log, FILENAME='/var/opt/mssql/host_data/${DB_NAME}.ldf')"
-    if [ $? -eq 0 ]; then
+    if test -f /var/opt/mssql/host_data/${DB_DIRECTORY}/${DB_NAME}.mdf; then
+        echo "Restoring from .mdf/.ldf"
+        /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P "$SA_PASSWORD" -Q "IF NOT EXISTS (SELECT * FROM sys.databases WHERE name = '${DB_NAME}') CREATE DATABASE ${DB_NAME} ON (NAME=${DB_NAME}_data, FILENAME='/var/opt/mssql/host_data/${DB_DIRECTORY}/${DB_NAME}.mdf') LOG ON (NAME=${DB_NAME}_log, FILENAME='/var/opt/mssql/host_data/${DB_DIRECTORY}/${DB_NAME}.ldf') FOR ATTACH;"
+        let result=$?
+    else
+        /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P "$SA_PASSWORD" -Q "IF NOT EXISTS (SELECT * FROM sys.databases WHERE name = '${DB_NAME}') CREATE DATABASE ${DB_NAME} ON (NAME=${DB_NAME}_data, FILENAME='/var/opt/mssql/host_data/${DB_DIRECTORY}/${DB_NAME}.mdf') LOG ON (NAME=${DB_NAME}_log, FILENAME='/var/opt/mssql/host_data/${DB_DIRECTORY}/${DB_NAME}.ldf')"
+        let result=$?
+    fi
+    if [ $result -eq 0 ]; then
         echo "Creating database completed"
         break
     else
         echo "Creating database. Not ready yet..."
         sleep 1
     fi
-done
\ No newline at end of file
+done
```

Comment:
I think this needs to be fixed on all three bash files... remove the empty last line. I know this empty line-ending can trip up on some systems. That's why we've added following in .editorConfig:

```
# Bash Files
[*.sh]
end_of_line = lf
insert_final_newline = false
```

Reply:
@JohanPetersson Fixed in [5d0b43e](https://github.com/episerver/content-templates/pull/47/commits/5d0b43ee2a5dcd7bccd2e4e3ffb0a5398688f3fd)